### PR TITLE
Add to app collection

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#7895

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,3 +80,48 @@ workflows:
           filters:
             tags:
               only: /^v.*/
+
+      # deploy to aws installations (only tags)
+      - architect/push-to-app-collection:
```
Comment (kopiczko) [inline .circleci/config.yml:85]:
```suggestion
      - architect/push-to-app-collection:
          context: "architect"
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -80,3 +80,48 @@ workflows:
...
+          requires:
+            - push-app-operator-to-control-plane-app-catalog
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
+
+      # deploy to azure installations (only tags)
+      - architect/push-to-app-collection:
```
Comment (kopiczko) [inline .circleci/config.yml:100]:
```suggestion
      - architect/push-to-app-collection:
          context: "architect"
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -80,3 +80,48 @@ workflows:
...
+          requires:
+            - push-app-operator-to-control-plane-app-catalog
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
+
+      # deploy to kvm installations (only tags)
+      - architect/push-to-app-collection:
```
Comment (kopiczko) [inline .circleci/config.yml:115]:
```suggestion
      - architect/push-to-app-collection:
          context: "architect"
```

**Comment by tomahawk28** [comment]
@rossf7 I created below script, WDYT? There are no configmap & secret but it would work. 
```
#!/bin/bash
l=$(opsctl list installations | sed '1d' | awk '{ print $1 }')
for t in $l; do
	echo ${t}
cat <<EOF | kubectl create -f -
apiVersion: application.giantswarm.io/v1alpha1
kind: Chart
metadata:
  annotations:
    chart-operator.giantswarm.io/force-helm-upgrade: "true"
  finalizers:
  - operatorkit.giantswarm.io/chart-operator-chart
  generation: 1
  labels:
    chart-operator.giantswarm.io/version: 1.0.0
    giantswarm.io/managed-by: app-operator
  name: app-operator-unique
  namespace: giantswarm
spec:
  name: app-operator-unique
  namespace: giantswarm
  tarballURL: https://giantswarm.github.io/control-plane-catalog/app-operator-1.0.0.tgz
  version: 1.0.0
EOF
done
```

**Comment by rossf7** [comment]
@tomahawk28 Yes the script looks fine. I don't think the YAML needs the generation field but it does no harm.

I have a concern about China as it will pull the image from Quay. But once app-operator processes its own CR it will use Aliyun again.

**Comment by tomahawk28** [comment]
I think it would pull from Aliyun if CP have a right value for it. https://github.com/giantswarm/app-operator/blob/1850d95ad79588c3882f46ddcde11c1dd68386b4/helm/app-operator/templates/deployment.yaml#L36

**Pull request merged**